docs/networking/networking.md

```diff
@@ -103,3 +103,46 @@ If you disable the built-in CCM and do not deploy and properly configure an exte
 ## Nodes Without a Hostname
 
 Some cloud providers, such as Linode, will create machines with "localhost" as the hostname and others may not have a hostname set at all. This can cause problems with domain name resolution. You can run K3s with the `--node-name` flag or `K3S_NODE_NAME` environment variable and this will pass the node name to resolve this issue.
+
+## Multicluster CIDR (Experimental)
+
+:::info Version Gate
+
+Experimental as of v1.26.3+k3s1
+
+:::
+
+:::caution Warning
+The network policy controller could not work properly when this flag is enabled.
+:::
+
+From `v1.26` Kubernetes introduced Multicluster CIDR as an alpha feature. This feature can be enabled on K3s server with the `--multi-cluster-cidr` flag.
+The `clustercidr` resources will be visible through the API and `kubectl` (the CIDR configured with `--cluster-cidr` is defined as the default one).
+
+A new `clustercidr` can be defined as follow:
+
+```
+apiVersion: networking.k8s.io/v1alpha1
+kind: ClusterCIDR
+metadata:
+  name: new-cidr
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          -  "worker2"
+  perNodeHostBits: 8
+  ipv4: 10.247.0.0/16
+```
+
+The nodes that match the `nodeSelector` will use a CIDR from the new defined resource.
```

Comment:
I think we can clarify that it is the Pod CIDR that is allocated to avoid confusion later.